docs/source/using/interacting-with-openshift.adoc

```diff
@@ -163,15 +163,18 @@ If you get the error `The specified patch need to be a valid JSON.` when you run
 you need to modify the above command depending on your Operating System,
 your shell environment and its interpolation behavior.
 
-For example, if you use PowerShell on Windows 10 modify the above command to:
+For example, if you use PowerShell on Windows 7 or 10 modify the above command to:
+
 ....
-$  .\minishift.exe --% openshift config set --patch "{\"routingConfig\": {\"subdomain\": \"192.168.99.101.nip.io\"}}"
+$ minishift openshift config set --patch '{\"corsAllowedOrigins\": [\".*\"]}'
 ....
 
 If you use Command Prompt you may need to use:
+
 ....
-minishift.exe openshift config set --patch "{\"routingConfig\": {\"subdomain\": \"192.168.99.101.nip.io\"}}"
+$ minishift openshift config set --patch "{\"corsAllowedOrigins\": [\".*\"]}"
```

Comment:
Nitpicking, the original doc had `minishift.exe` as it is for Windows. However `minishift` also works.